tests/unit_tests/test_coverage.py

```diff
@@ -20,10 +23,62 @@ def fixture_metagenome(variables, tmp_path):
     return fpath.as_posix()
 
 
-# TODO: Create fixtures for each set of data.
-# Then group fixtures into a group and indirectly parametrize group
-# i.e. alignments.sam, alignments.bam, alignments.bed, fwd_reads.fq, rev_reads.fq
-# The external tools could be run or we could monkeypatch these.
+@pytest.fixture(name="sam_alignment")
+def fixture_alignment_sam(variables, tmp_path):
+    coverage_test_data = variables["coverage"]
+    outlines = coverage_test_data["sam"]
+    fpath = tmp_path / "records.sam"
+    with open(fpath, "w") as fh:
+        fh.write(outlines)
+    return fpath.as_posix()
+
+
+@pytest.fixture(name="bam_alignment")
+def fixture_alignment_bam(variables, sam_alignment, tmp_path):
+    bam_fpath = tmp_path / "records.bam"
+    samtools.sort(sam=sam_alignment, bam=bam_fpath)
+    return bam_fpath.as_posix()
+
+
+@pytest.fixture(name="bed_alignment")
+def fixture_alignment_bed(variables, tmp_path):
+    coverage_test_data = variables["coverage"]
+    alignment_records = pd.read_json(coverage_test_data["bed"])
+    fpath = tmp_path / "records.bed"
+    alignment_records.to_csv(fpath, sep="\t", header=True, index=False)
+    return fpath.as_posix()
```

Comment:
`bedtools.parse` uses the `pd.read_csv` method to read in the table. `pd.read_csv` can handle Pathlib objects. So I'm not entirely sure, but I do not think you need to use `fpath.as_posix()` but could simply return `fpath`. 
 Otherwise, see comments below about `fpath.as_posix()` -> `str(fpath)`